api-documentation.md

```diff
@@ -27,7 +27,7 @@ This call requests a new survey from Pollfish servers
 Structure:
 
 ```shell
-https://wss.pollfish.com/v2/device/register/true?json={}&dontencrypt=true
+https://wss.pollfish.com/v2/device/register/true?json={}&dontencrypt=true&sig="a sig"
```

Comment:
please provide both options, do not replace the old one..

Reply:
You mean 2 url, both we sig and without it?

Reply:
keep the one as it is and add an alternative with sig below please